aldryn_faq/models.py

```diff
@@ -194,7 +209,9 @@ class QuestionsPlugin(models.Model):
     )
 
     def get_queryset(self):
```

Comment:
it's quite dangerous to override the ``get_queryset`` method to filter out stuff. Keep in mind that one thing is the queryset of objects that are user facing and another is the objects that show up on the admin and internally objects that are available for django when doing things like get_or_create()